src/Common/BlockEditor/Blocks/QuoteBlock.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Common\BlockEditor\Blocks;
+
+final class QuoteBlock extends AbstractBlock
+{
+    public function getConfig(): array
+    {
+        return [
+            'class' => 'BlockEditor.blocks.Quote',
+        ];
+    }
+
+    public function getValidation(): array
+    {
+        return [
+            'text' => [
+                'type' => 'string',
+                'required' => true,
+                'allowedTags' => 'b,i,a[href]',
+            ],
+            'caption' => [
+                'type' => 'string',
+                'required' => true,
+                'allowedTags' => 'b,i,a[href]',
+            ],
+            'alignment' => [
+                'type' => 'string',
```

Comment:
I would add
        "canBeOnly": ["left", "center"]

Reply:
fix in #5cf5ca1